apps/electron/main.js

```diff
@@ -138,6 +138,11 @@ function createWindow() {
   });
 
   win.setMenu(null);
+
+  win.webContents.setWindowOpenHandler(({ url }) => {
+    shell.openExternal(url);
+    return { action: 'deny' };
```

Comment:
it is what opens the auth process in the browser and not a new electron window

Reply:
That is the exact same that the other function though? If it doesn't achieve anything new we should reuse it

Reply:
As far as I can tell, they're not exactly the same . `send-to-link` needs to be called via IPC like [here](https://github.com/cpvalente/ontime/blob/d6aef5ea316d2a2a82a8fac5ebd325f88ddf0ad3/apps/client/src/common/utils/linkUtils.js#L15) whereas setWindowOpenHandler catches all `window.open()` calls as well as links with `target=_blank`
Maybe it would make more sense to use the new one instead of the existing everywhere since we shouldn't need the if statement in the js anymore.

Reply:
That is a good observation.

It can be true that we can make the app-wide decision that we will never want to open a link in Electron.
If so, we can use the new function, but we would then remove the old one.
We should not have both